packages/pilot/package.json

```
@@ -9,8 +9,8 @@
     "cockpit": "0.0.1",
     "cpf_cnpj": "0.2.0",
     "emblematic-icons": "0.6.1",
-    "former-kit": "1.4.0",
-    "former-kit-skin-pagarme": "1.2.0",
+    "former-kit": "1.5.0",
```

pode atualizar pra 1.5.1 já =)
```suggestion
    "former-kit": "1.5.1",
```

---

Boa @rsmelo!